pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -719,8 +709,10 @@ private BrokerResponseNative handleRequest(long requestId, String query,
         TimeUnit.MILLISECONDS);
 
     // Extract source info from incoming request
-    logBrokerResponse(requestId, query, requestContext, tableName, numUnavailableSegments, serverStats, brokerResponse,
-        totalTimeMs, requesterIdentity);
+    _queryLogger.log(
```

Comment:
I am guessing (hoping actually) the log response format / structure is not changed by this PR ? I ask because we have few internal scripts / tools for debug etc that parse the broker query log and so are aware of the log pattern / regex.

Reply:
yes this is correct, I triple/quadruple checked (and going forward there's a test to make sure it doesn't)!